doc/release-notes/8424-signposting.md

```diff
@@ -0,0 +1,42 @@
+# Signposting for DV 5.9
+
+This branch adds [Signposting](https://signposting.org/) support to DV 5.9.
+
+There are 2 Signposting profile levels, level 1 and level 2. In this implementation, 
+ * level 1 links are shown in 
+HTTP header, which can be fetched by `curl -I https://domain/link-to-article`. 
+ * The level 2 linkset can be fetched by visiting the dedicated linkset page for 
+   that artifact. The link can be seen in level 1 links with key name `rel="linkset"`.
+
+The configuration is stored as JSON string in the `Bundle.properties` file, key name is
+`signposting.configuration.SignpostingConf`. Please see a sample configuration below with explaination for each of the
+config items.
+
+```json
+{
+  "license": {
+    "CC0": "https://creativecommons.org/licenses/cc0/"
+  },
+  "describedby": {
+    "doi": "https://doi.org/",
```

Comment:
This is to make sure in case people have a dataverse instance with mixed id types, they can get proper url, `https://doi.org` in this case, by using `describedByJson.getString(ds.getProtocol())` 
If there is a better way to get per dataset `identifier` and `identifier type`, please do let me know. Thanks!